NumLib/Fem/CoordinatesMapping/NaturalCoordinatesMapping.cpp

```diff
@@ -96,6 +96,28 @@ computeMappingMatrices(
 {
 }
 
+static void checkJacobianDeterminant(const double detJ)
+{
+    if (detJ <= .0)
+    {
+        if (detJ == .0)
+        {
+            OGS_FATAL(
+                "det J is zero. Please check whether:\n"
+                "\t element nodes may have the same coordinates,\n"
+                "\t or the coordinates of all nodes are not given in x "
+                "component for 1D problem,\n"
+                "\t or the coordinates of all nodes are not given in x-y "
```

Comment:
by 'x component' and 'x-y component' you mean the x-axis and x-y-plane?

Reply:
Yes, it is (as your changes in your PR).